src/locales/en/en.json

```diff
@@ -7,5 +7,13 @@
   "ORGANIZATION":"Organization",
   "ORGANIZATION_NAME":"Organization Name",
   "CHANNEL_NAME":"Channel Name",
-  "CHANNEL_ID":"Channel ID"
+  "CHANNEL_ID":"Channel ID",
+  "FIRST_NAME":"First Name",
```

Comment:
use ENTITY NAME / VIew name as prefix to all newly added lang constants 

eg   FIRST_NAME =>   USER_FIRST_NAME / USER_LIST_FIRST_NAME